test/simplifyExpression/basicsSearch/basicTest.js

```diff
@@ -14,15 +14,16 @@ describe('basic rules', function() {
       ['----5','--5'],
     ];
 
-    tests.forEach(t => testSimplify(t[0], t[1], basics.RESOLVE_DOUBLE_NEGATION, ChangeTypes.RESOLVE_DOUBLE_NEGATION));
+    tests.forEach(t => testSimplify(t[0], t[1], basics.RESOLVE_DOUBLE_NEGATION,
+                                    ChangeTypes.RESOLVE_DOUBLE_NEGATION));
   });
 
   describe.skip('rearrange coefficient', function() {
     const tests = [
-        ['y^3 * 5', '5 y^3'],
-        ['yz * 3', '3 yz'],
-        // TODO: handle this case better
-        //['3x^2 * 5', '5 (3 x^2)']
+      ['y^3 * 5', '5 y^3'],
+      ['yz * 3', '3 yz'],
+      // TODO: handle this case better
```

Comment:
Waiting for the `math-node` query functions PR to be merged since `rearrangeCoefficient` requires one of the methods.